### DISCUSSION TITLE
Close() function not available when File opened with excelize.OpenReader()

### DISCUSSION THREAD

**Discussion by dwschulze** [qax-os/excelize#1178] [General]
I get an `*excelize.File` with

	`f, err := excelize.OpenReader(file)`

where the file argument is a `multipart.File`.  If I try to do

    `defer f.Close()`

I get this error:

     _f.Close undefined (type *excelize.File has no field or method Close)_

If I get an `*excelize.File` with

    `f, err := excelize.OpenFile(fileName)`

then I can call `.Close()` on it.

How do I assure that the `excelize.File` obtained from `OpenReader()` gets closed?

**Comment by xuri** [comment]
Thanks for your issue, this issue is duplicated with #1025, #1042, and #1091, the `Close` function has been introduced since v2.5.0, if you're using the version early than v2.5.0, needn't call this function.
